romana-install/create_hosts.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: localhost
+  tasks:
+   - name: Create path for stack files
+     file: path="./{{ stack_data_dir }}" state="directory"
+
+# Only the target platform will be created
+- include: create_hosts.aws.yml
```

Comment:
We can put a "when:" clause on those includes, so that we only include stuff that's really needed and don't have to do the "when" in the include.